docker-compose.yml

```
@@ -68,7 +68,7 @@ services:
      - ./build:/mapping
      - cache:/cache
   import-sql:
-    image: "openmaptiles/import-sql:0.8"
+    image: "openmaptiles/import-sql:0.9"
```

it seems that the docker hub has been updated with the latest import-sql code (https://hub.docker.com/r/openmaptiles/import-sql/tags) but the version jumped to `1.0` instead of `0.9`

Tested with the change above and it worked well for me

```suggestion
    image: "openmaptiles/import-sql:1.0"
```